modules/installation-and-upgrade/pages/proxy-registration.adoc

```diff
@@ -33,12 +33,12 @@ image::proxy-key.png[]
 
 . From the [guimenu]``Child Channels`` listing select the recommended channels by clicking the ``include recommended`` icon:
 +
-* SLE-Module-Basesystem15-SP5-Pool
-* SLE-Module-Basesystem15-SP5-Updates
-* SLE-Module-Server-Applications15-SP5-Pool
-* SLE-Module-Server-Applications15-SP5-Updates
-* SLE-Module-SUSE-Manager-Proxy-4.3-Pool
-* SLE-Module-SUSE-Manager-Proxy-4.3-Updates
+* SLE-Module-Basesystem15-{sp-base}-Pool
+* SLE-Module-Basesystem15-{sp-base}-Updates
+* SLE-Module-Server-Applications15-{sp-base}-Pool
+* SLE-Module-Server-Applications15-{sp-base}-Updates
+* SLE-Module-SUSE-Manager-Proxy-{productnumber}-Pool
```

Comment:
```suggestion
* SLE-Module-SUSE-Manager-Proxy-4.3-Pool
```

Reply:
Reverting back as per @jcayouette comment (see below). I agree with his POV, we need to have a clean issue and PRs dealing with this.